src/modules/hmis/ClientIdEncoder.ts

```diff
@@ -0,0 +1,64 @@
+import CryptoJS from 'crypto-js';
+import { startOfToday } from 'date-fns';
+
+export default class ClientIdEncoder {
```

Comment:
Can you add some unit tests for this class @ZachCMP ?

Reply:
done